runelite-client/src/main/java/net/runelite/client/plugins/inventorytags/InventoryTagsOverlay.java

```diff
@@ -54,11 +54,22 @@ public void renderItemOverlay(Graphics2D graphics, int itemId, WidgetItem itemWi
 		if (group != null)
 		{
 			final Color color = plugin.getGroupNameColor(group);
+			final InventoryTagsConfig.DisplayMode displayMode = plugin.getDisplayMode();
```

Comment:
Import `net.runelite.client.plugins.inventorytags.InventoryTagsConfig.DisplayMode` instead of using inline qualifying of `DisplayMode`.